services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/PathNodePayload.java

```diff
@@ -41,19 +42,21 @@
      * Input port number.
      */
     @JsonProperty("input_port")
-    private int inputPort;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private Integer inputPort;
```

Comment:
what's the reason for Object instead of primitive here?

Reply:
answered above